### PR TITLE
Add resp msg in create conneciton obj api

### PR DESCRIPTION
**Pull request by Roshan-Mehta**
### Changes

Create connection obj API was returning nil in every case, it was causing issue in wasm. In this PR, I have added some response message to API. 

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/blobber/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- 0chain:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/blobbercore/blobberhttp/response.go [hunk 1]
```diff
@@ -22,6 +22,13 @@ type UploadResult struct {
 	UploadOffset int64 `json:"upload_offset"`
 }
 
+// swagger:model ConnectionResult
+type ConnectionResult struct {
+	AllocationRoot string `json:"allocation_root"`
+	ConnectionID   string `json:"connection_id"`
+	Success        bool   `json:"success"`
```
Comment (dabasov) [inline code/go/0chain.net/blobbercore/blobberhttp/response.go]: looks like we don't use this field, let's remove it

**Review by dabasov — CHANGES_REQUESTED** [review]

File: code/go/0chain.net/blobbercore/blobberhttp/response.go (hunk 1 above)
Comment (Roshan-Mehta) [reply code/go/0chain.net/blobbercore/blobberhttp/response.go]: Which one?

Comment (Roshan-Mehta) [reply code/go/0chain.net/blobbercore/blobberhttp/response.go]: Done

**Review by dabasov — APPROVED** [review]
lgtm

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | master  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | fix/send-resp-create_connection  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Pull request merged**